index.js

```diff
@@ -0,0 +1,16 @@
+function splitNumber(number){
+    return number.toString().split('.');
+    }
+  
+  function handleHours(){
+      const inputNumber = document.getElementById("input-number");
+      const number = inputNumber.value;
```

Comment:
Só por :nail_care: mas voce pode resumir isso pra :

```suggestion
      const time =  document.getElementById("input-number").value;
```